content/updates/2021-03/5-Events-API-logs.md

```diff
@@ -0,0 +1,26 @@
+---
+pim_announcement_img: ./img/event-logs-screen.png
+pim_announcement_alt_img: "Events API log screen"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Browse your Events API logs
+::: meta-data type="New" features="Connectivity" available="late March" in="EE,GE" link-to-doc="../articles/manage-event-subscription.html#debug"
+
+To ease the debugging of your Events API-based connections, we introduce a brand new monitoring screen to display the Events API logs, whether they are just info, notices, warnings, or even errors. A great addition to help you find root causes of synchronization problems and also, without a doubt, an appreciated time-saver to understand what happened on your connections!
+
+You can find this new screen in the `Event logs` menu of each connection. Note that you can use the date and level filters to narrow down the selection of displayed logs.
```

Comment:
```suggestion
You can find this new screen under the `Event logs` menu of each connection. Note that you can use the date and level filters to narrow down the selection of displayed logs.
```

Reply:
@camillebourjade Is is ok to use "under" here?

Reply:
I would stick to 'in'